### PR TITLE
Collection list command added

### LINKED ISSUE #37: Getter function for Collection List
# Rationale

This feature is required for the admin or user to see the list of collections and their respective info as provided by the Structs contract.

# Implementation

The implementation would be the same as #35

### PR DESCRIPTION
**Pull request by shekhar2807**
# Description

collectionList command added.
It returns details like  power, Id, name etc of all the collections.

Fixes #37 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test 1: When collectionList executes properly
- [x] Test 2: When there is a error fetching collection list

**Test Configuration**:
* Contracts version: v0.1.8
* Hardware: Macbook M1 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ashish10677 — APPROVED** [review]
lgtm

File: cmd/collectionList_test.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+			}
+
+			GetCollectionsMock = func(*ethclient.Client) ([]bindings.StructsCollection, error) {
+				return tt.args.collectionList, tt.args.collectionListErr
+			}
+			err := utilsStruct.GetCollectionList(tt.args.client)
+
+			if err == nil || tt.wantErr == nil {
+				if err != tt.wantErr {
+					t.Errorf("Error for jobList function, got = %v, want = %v", err, tt.wantErr)
```
Comment (Yashk767) [inline cmd/collectionList_test.go:73]: It should be "collectionList" instead of "jobList" in error log

File: cmd/collectionList_test.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+				}
+			} else {
+				if err.Error() != tt.wantErr.Error() {
+					t.Errorf("Error for jobList function, got = %v, want = %v", err, tt.wantErr)
```
Comment (Yashk767) [inline cmd/collectionList_test.go:77]: Here as well ,should be "Error for collectionList function"

**Review by Yashk767 — CHANGES_REQUESTED** [review]

**Review by Yashk767 — APPROVED** [review]
LGTM

**Pull request merged**